src/main/java/appeng/items/tools/powered/ToolWirelessTerminal.java

```diff
@@ -77,6 +122,18 @@ public void addCheckedInformation( final ItemStack stack, final EntityPlayer pla
 	{
 		super.addCheckedInformation( stack, player, lines, displayMoreInfo );
 
+		if( isLinked( stack ) )
+		{
+			lines.add(GuiText.Linked.getLocal());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ')' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '(' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)